R/pkg/NAMESPACE

```diff
@@ -10,6 +10,10 @@ export("sparkR.init")
 export("sparkR.stop")
 export("print.jobj")
 
+# MLlib integration
+export("glm")
```

Comment:
Should we introduce a namespace for these MLlib APIs? For example, `ml.glm`, `ml.predict`

We may introduce lots of them in next release, then it's easy to have conflicts with other R library. Or we let's user to handle the conflict?